lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
+  require Logger
+
+  def deliver_now(email) do
+    ses_rate = Application.get_env(:hex_web, :ses_rate) |> String.to_integer
+
+    email
+    |> recipients
+    |> recipient_chunks(ses_rate)
+    |> Enum.each(fn recipients ->
+      HexWeb.Throttle.wait(HexWeb.SESThrottle, length(recipients))
+      HexWeb.Mailer.deliver_now(email)
```

Comment:
looking at this line, it seems to me we're sending the same (= same list of recipients) email N times, instead of sending to each recipient chunk as in the previous implementation